src/frontend/index.js

```diff
@@ -28,4 +37,29 @@ function runSampleProgram() {
     });
 }
 
-runSampleProgram();
\ No newline at end of file
+var backend;
+
+function initBackend() {
+    require(["jquery", "bootstrap", "backend"], function(be) {
+        backend = be;
+    });
+
+    require(["cm/lib/codemirror",
```

Comment:
1. I don't think this is part of "backend."
2. Please init this codemirror in a separate file at least.